eq-author/src/App/page/Routing/RoutingPage/RoutingEditor/RuleEditor/BinaryExpressionEditor/MultipleChoiceAnswerOptionsSelector.js

```diff
@@ -15,9 +15,16 @@ import Popover from "./CheckboxSelectorPopup";
 import CheckboxOptionPicker from "./CheckboxOptionPicker";
 import CheckboxChip from "./CheckboxChip";
 
+const answerConditions = {
+  UNANSWERED: "Unanswered",
+  ALLOF: "AllOf",
+  ANYOF: "AnyOf",
+};
+
 const MultipleChoiceAnswerOptions = styled.div`
   align-items: center;
   padding: 1em 0;
+  display: inline-flex;
```

Comment:
Unfortunately I think just doing this will mess up radio buttons when there are a lot of options. Try adding `flex-flow: row wrap;` to the css as well?